### ISSUE TITLE
`lookForAlignDelims` and `tabbing`?

### ISSUE THREAD

**Issue by anyong** [cmhughes/latexindent.pl#187]
According to [listing 13](https://latexindentpl.readthedocs.io/en/latest/sec-indent-config-and-settings.html#indentconfig-yaml-and-indentconfig-yaml), we should be able to add the `tabbing` environment to `lookForAlignDelims`. However, `tabbing` uses `\=` and `\>` for alignment, while `lookForAlignDelims` seems to only recognize `&`.

Is this a typo in the docs, or is there some other command I missed to align my `tabbing` delimeters `\=` and `\>`?

**Comment by cmhughes** [comment]
Thanks for this.

This isn't currently possible. If you can provide examples of before and
after, I can see what I can do.

**Comment by anyong** [comment]
Basically exactly the same as the alignment based on `&`, but in `tabbing` the first line has `\=` to set tabstops and subsequent lines use `\>` to align tabs. Here's an example of what I'd like it to accomplish?

Before:

```
\begin{tabbing}
    Tone \= 1 changes to Tone \= 7 \qquad Tone \= 5 changes to Tone \= 7 \\
    \> 2 \> 1 \> 7 \> 3 \\
    \> 3 \> 2 \> 8 \> 3 \\
    \> 4 \> 2 \\
\end{tabbing}
```

After:

```
\begin{tabbing}
    Tone \= 1 changes to Tone \= 7 \qquad Tone \= 5 changes to Tone \= 7 \\
         \> 2                 \> 1             \> 7                 \> 3 \\
         \> 3                 \> 2             \> 8                 \> 3 \\
         \> 4                 \> 2                                       \\
\end{tabbing}
```

![image](https://user-images.githubusercontent.com/6260469/73941872-fb307d80-4928-11ea-90bc-405eb00fbbc0.png)

**Comment by cmhughes** [comment]
That's great, thanks. Should be doable, but there's a few ahead of this. I'll let you know when I have an update, but it may well be a while.

**Comment by cmhughes** [comment]
This feature could be interesting for bibliography entries also:

```
@Article{a-key,
  author    = {Person One},
  title     = {A demo in lowercase},
  journal   = {Journal of Irreproducible Results},
  year      = {2020}
}
```

**Comment by cmhughes** [comment]
As of https://github.com/cmhughes/latexindent.pl/commit/e4495d42af43c54be6ad42c3d0c0e20ec76462a1 this is implemented in the `develop` branch, and documented. 

If you'd like to clone the `develop` branch, you should be able to test it. For your example, you might like the following YAML:
```
lookForAlignDelims:
   tabbing: 
    delimiterRegEx: '(\\(?:=|>))'
```

Thanks for suggesting this!

**Comment by cmhughes** [comment]
demonstration
-
Starting with
```
\begin{tabbing}
aa \=
bb \= cc \= dd \= ee \\
\>2\> 1 \> 7 \> 3 \\
\>3 \> 2\>8\> 3 \\
\>4 \>2 \\
\end{tabbing}
```
and the YAML
```
lookForAlignDelims:
   tabbing:
   delimiterRegEx: '(\\(?:=|>))'
```
gives
```
\begin{tabbing}
	aa \= bb \= cc \= dd \= ee \\
	   \> 2  \> 1  \> 7  \> 3  \\
	   \> 3  \> 2  \> 8  \> 3  \\
	   \> 4  \> 2              \\
\end{tabbing}
```

**Comment by cmhughes** [comment]
Implemented in `Version 3.8`, https://github.com/cmhughes/latexindent.pl/pull/190
